.gitignore

```diff
@@ -32,6 +32,8 @@ amplify/backend/amplify-meta.json
 amplify/backend/.temp
 build/
 dist/
+node_modules/
+aws-exports.js
```

Comment:
I think u need to delete these two lines so that aws-exports and node_moduels are available to aws with the git building & continuous integration option we have set up